core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -212,7 +211,7 @@ private[spark] class BlockManager(
   private val maxOnHeapMemory = memoryManager.maxOnHeapStorageMemory
   private val maxOffHeapMemory = memoryManager.maxOffHeapStorageMemory
 
-  private val externalShuffleServicePort = StorageUtils.externalShuffleServicePort(conf)
+  private[spark] val externalShuffleServicePort = StorageUtils.externalShuffleServicePort(conf)
```

Comment:
Why did the scope change here?

Reply:
It's required by `val port = blockManager.externalShuffleServicePort` within `ShuffleBlockFetcherIterator`.